settings/Memory_Z3Ram_64.txt

```diff
@@ -1,2 +1,3 @@
 BeneathASteelSkyCD32
-onEscapee
\ No newline at end of file
+onEscapee
+TZer0DemoAGA
```

Comment:
whdload.de suggests this needs 'about 28 meg for preload option' - i can add a 32 meg Z3 option for this?

The main reason to do this, would be the potential size of any save-state files

BeneathASteelSkyCD32 on the other hand, probably needs more than this...

Reply:
Yes, I had mine set at 32meg but there wasn't a file for it !